WordPress/Classes/Utility/OverlayFrequencyTracker.swift

```diff
@@ -76,6 +76,9 @@ class OverlayFrequencyTracker {
         guard let lastSavedGenericDate = lastSavedGenericDate else {
             return true // First overlay ever
         }
+        if frequencyConfig.generalInDays < 0 {
+            return false // Negative frequency means overlay should be shown once
+        }
```

Comment:
I agree that it's not 100% clear. However, I'm unsure how to enhance this without a big rewrite. Using an enum means I need to refactor `FrequencyConfig` and parts of `JetpackFeaturesRemovalCoordinator`.

One idea for an easy win is to add computed variables inside `FrequencyConfig` that check if the frequency is negative. It could be named something like `shouldShowOnce`. Let me know what you think, or if you have other ideas!

Reply:
I understand. Feel free to proceed as you see fit. What you suggested makes sense to me although it isn't a must-have.